.ci/scripts/go-plus/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -21,9 +22,10 @@ pushd $local_path
 
 # Only skip tests if we can tell for sure that no go files were changed
 echo "Checking for modified go files"
+git fetch origin $old_branch:$old_branch --depth 1
```

Comment:
```suggestion
# Fetch the latest commit in the old branch, associating them locally
# This will let us compare the old and new branch by name on the next line
git fetch origin $old_branch:$old_branch --depth 1
```